server/WebAPI/Startup.cs

```diff
@@ -39,6 +45,25 @@ public void ConfigureServices(IServiceCollection services)
                                   });
             });
 
+            services.AddDbContext<RepositoryContext>(options => 
+                options.UseSqlServer(Configuration.GetConnectionString("mssqlConnection")));
+            services.AddSingleton(Configuration);
+            services.AddScoped<UserRepository>();
+            services.AddScoped<UserService>();
+            services.AddSingleton<AuthService>();
```

Comment:
just curoius, why do we want auth service as sigleton while others are scoped?

Reply:
That`s because now AuthService doesn`t use any Scoped instances unlike UserService